data/formats-data.js

```diff
@@ -4599,7 +4606,7 @@ exports.BattleFormatsData = {
 		tier: "RU",
 	},
 	woobat: {
-		randomBattleMoves: ["calmmind", "psychic", "airslash", "gigadrain", "roost", "heatwave", "storedpower"],
+		randomBattleMoves: ["calmmind", "roost", "heatwave", "storedpower"],
```

Comment:
keep psychic as an option - immediate power is sometimes favoured over stored power's long term effect.

Reply:
ordinarily id agree with your argument but storedpower outdoes psychic after 1 cm thanks to simple